imports/ui/record-game/RecordJapaneseGame.js

```diff
@@ -1034,18 +1059,25 @@ function push_restart_hand(template) {
 };
 
 function push_mistake_hand(template) {
-    var loserWind = GameRecordUtils.playerToDirection(Session.get("round_loser"));
-    var eastDelta = mistake_delta(Constants.EAST, loserWind);
-    var southDelta = mistake_delta(Constants.SOUTH, loserWind);
-    var westDelta = mistake_delta(Constants.WEST, loserWind);
-    var northDelta = mistake_delta(Constants.NORTH, loserWind);
-
-    if          (loserWind == Constants.EAST)  Session.set("eastMistakeTotal",  Number(Session.get("eastMistakeTotal"))  + 1);
-    else if (loserWind == Constants.SOUTH) Session.set("southMistakeTotal", Number(Session.get("southMistakeTotal")) + 1);
-    else if (loserWind == Constants.WEST)  Session.set("westMistakeTotal",      Number(Session.get("westMistakeTotal"))  + 1);
-    else if (loserWind == Constants.NORTH) Session.set("northMistakeTotal", Number(Session.get("northMistakeTotal")) + 1);
+    let loserWind = GameRecordUtils.playerToDirection(Session.get("round_loser"));
+    let handDeltas = HandScoreCalculator.jpn.mistake_delta(loserWind);
+
+    if (loserWind == Constants.EAST) {
+        Session.set("eastMistakeTotal", Number(Session.get("eastMistakeTotal")) + 1);
+    } else if (loserWind == Constants.SOUTH) {
+        Session.set("southMistakeTotal", Number(Session.get("southMistakeTotal")) + 1);
+    } else if (loserWind == Constants.WEST) {
+        Session.set("westMistakeTotal", Number(Session.get("westMistakeTotal")) + 1);
+    } else if (loserWind == Constants.NORTH) {
+        Session.set("northMistakeTotal", Number(Session.get("northMistakeTotal")) + 1);
+    }
 
```

Comment:
Just thinking that nested 'if-else' can be a bit unreadable, and there isn't an 'else' condition at the end of it all anyways.  Why not use cases (or just 'if's)?

Reply:
Just ifs wouldn't work because you only want it going down one path of execution. In this case, a switch-case statement makes more sense, unless the breaks damage readability lol

Reply:
I want to clean code up in a later PR.  This is just to not have one-line blocks.

Reply:
Gotcha